content/docs/start/experiments/experiment-tracking.md

```diff
@@ -86,6 +96,8 @@ with Live(save_dvc_exp=True) as live:
         for metric_name, value in metrics.items():
             live.log_metric(metric_name, value)
         live.next_step()
+
+    live.log_artifact("model.pkl", type="model")
 ```
```

Comment:
Feels like we need one sentence at the end of this section (below these examples) to explain log_artifact (saves artifacts with dvc and enables managing them with model registry).

Reply:
edited paragraph few lines below to explain this.